pepper-apis/ddp-common/src/main/java/org/broadinstitute/ddp/util/ConfigManager.java

```diff
@@ -44,7 +44,8 @@ public class ConfigManager {
 
         TYPESAFE_CONFIG_FILE = Optional.ofNullable(configFileName).map(File::new).orElse(null);
         if (TYPESAFE_CONFIG_FILE != null && !TYPESAFE_CONFIG_FILE.exists()) {
-            throw new DDPException("The config file " + TYPESAFE_CONFIG_FILE.getAbsolutePath() + " doesn't exist");
+            throw new DDPException("The config file " + TYPESAFE_CONFIG_FILE.getAbsolutePath() + " doesn't exist.  If you're using a "
```

Comment:
Added more explicit suggestion here to check for absolute path of conf file, since I failed to do this and it took me a while to figure out that this was the source of my issue.